src/test/java/io/github/glytching/junit/extension/random/RandomBeansExtensionFieldTest.java

```diff
@@ -34,6 +34,12 @@
 @ExtendWith(RandomBeansExtension.class)
 public class RandomBeansExtensionFieldTest {
 
+  @Random private static String anyStaticString;
+
+  // gather the random values to facilitate assertions on the not distinct-ness of the value supplied to
+  // each test
+  private static Collection<String> staticStringCollection = new HashSet<>();
```

Comment:
Could you rename this to `anyStaticStrings` to mirror `anyStrings`?

Reply:
Thanks! Done.